prime-router/build.gradle.kts

```diff
@@ -449,16 +449,9 @@ tasks.register("testEnd2End") {
     finalizedBy("primeCLI")
 }
 
-tasks.register("testEnd2EndUP") {
```

Comment:
Is this task no longer necessary?

Reply:
Not sure why it was removed. I recall discussion around merging the two -- testEnd2End and testEnd2EndUP -- but I haven't found anything concrete there so I've added it back in for now.